gitlab/cli.py

```diff
@@ -395,37 +400,39 @@ def main() -> None:
     gitlab_resource = args.gitlab_resource
     resource_action = args.resource_action
     skip_login = args.skip_login
+    unmask_credentials = args.unmask_credentials
 
     args_dict = vars(args)
     # Remove CLI behavior-related args
     for item in (
-        "gitlab",
+        "api_version",
         "config_file",
-        "verbose",
         "debug",
+        "fields",
+        "gitlab",
         "gitlab_resource",
-        "resource_action",
-        "version",
+        "job_token",
+        "oauth_token",
         "output",
-        "fields",
+        "pagination",
+        "private_token",
+        "resource_action",
         "server_url",
+        "skip_login",
         "ssl_verify",
         "timeout",
-        "api_version",
-        "pagination",
+        "unmask_credentials",
         "user_agent",
-        "private_token",
-        "oauth_token",
-        "job_token",
-        "skip_login",
+        "verbose",
+        "version",
     ):
         args_dict.pop(item)
     args_dict = {k: _parse_value(v) for k, v in args_dict.items() if v is not None}
 
     try:
         gl = gitlab.Gitlab.merge_config(vars(options), gitlab_id, config_files)
         if debug:
-            gl.enable_debug()
+            gl.enable_debug(mask_credentials=not unmask_credentials)
```

Comment:
To go with the above:

```suggestion
            gl.enable_debug(mask_credentials=mask_credentials)
```

Reply:
Done